### ISSUE TITLE
Add landmark elements to ensure all content is contained within landmarks

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#549]
Related to #548.

Not all page content is contained by a landmark element. In addition to adding the `main` landmark as noted in #548, we should evaluate the structure of each distinct view and figure out how to update each view with HTML5 sectioning elements and/or ARIA roles to ensure we have a thought-through landmark strategy. This will improve navigational accessibility.

**Comment by jvine** [comment]
#622 and #624 cover parts of this issue

**Comment by ggeisler** [comment]
Superseded by #820 and related tickets.
